astropy/stats/src/compute_bounds.h

```diff
@@ -1,8 +1,8 @@
 #ifndef __COMPUTE_BOUNDS_H__
 #define __COMPUTE_BOUNDS_H__
 
-void compute_sigma_clipped_bounds(double buffer[], int count, int use_median,
+void compute_sigma_clipped_bounds(double buffer[], int count, int use_median, int use_mad_std,
```

Comment:
Update the names of `buffer` and `buffer2` here too

Reply:
Fixed